### PR TITLE
Fixes #1172 reduce tracking pixel size back to 1px 1px

### LINKED ISSUE #1172: Tracking Pixel from Medium Posts cause display issues
**What happened**:
In posts aggregated from Medium, a tracking pixel is added at the end of <content:encoded> tags in the rss feed. We display this image on Telescope with a width of 100%, giving the appearance of significant extra padding.
This is an issue when viewing Telescope using Chrome or Chromium based browsers, like Samsung internet and Edge.

Example:
Mobile, Chrome:
image
image
Desktop Chrome:
image

**What should have happened**:
Since the image is not a part of the original post, it should not appear on Telescope.
image
image

**How to reproduce it (as precise as possible)**:
Go to Telescope using Chrome, either desktop or mobile, look for a blog post originally posted on Medium.

**Anything else we need to know?**: 
Firefox does not have this issue, the image is still is in the code, but it cannot load it

**Environment**:

- OS: Windows
- Browser: Chrome, Samsung Internet, Edge

### PR DESCRIPTION
**Pull request by cindyledev**
## Issue This PR Addresses
Fixes #1172 

## Type of Change

- [ ] **Bugfix**: Change which fixes an issue
- [ ] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [x] **UI**: Change which improves UI

## Description

This PR reduces the size of the tracking pixels found in Medium.com blogs to 1px 1px

## Before
Screenshot 2020-10-20 122822

## After
Screenshot 2020-10-20 122919

## Checklist

- [ ] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

**Review by agarcia-caicedo — DISMISSED** [review]

File: src/frontend/src/components/Post/telescope-post-content.css [hunk 1]
```diff
@@ -41,6 +41,12 @@
   vertical-align: center;
 }
 
+.telescope-post-content img[src*='https://medium.com/_/stat']
+{
```
Comment (humphd) [inline src/frontend/src/components/Post/telescope-post-content.css:45]: Move this up to 44 with the selector please.

**Review by humphd — COMMENTED** [review]
Should we do this, or should we strip it out when we santize the HTML at feed parsing time with https://www.npmjs.com/package/sanitize-html?

**Comment by cindyledev** [comment]
> Should we do this, or should we strip it out when we santize the HTML at feed parsing time with https://www.npmjs.com/package/sanitize-html?

I had a look at sanitize HTML and couldn't figure out how to block certain attributes or in our case, part of a URL. It seems to have allow rules and block everything else that's not listed in the allow rules.

File: src/frontend/src/components/Post/telescope-post-content.css (hunk 1 above)
Comment (cindyledev) [reply src/frontend/src/components/Post/telescope-post-content.css]: sorry, auto save on format changed in my VS Code for some reason

**Review by humphd — APPROVED** [review]

**Review by humphd — CHANGES_REQUESTED** [review]
```
Checking formatting...
[warn] src/frontend/src/components/Post/telescope-post-content.css
[warn] Code style issues found in the above file(s). Forgot to run Prettier?
```

**Comment by cindyledev** [comment]
> ```
> Checking formatting...
> [warn] src/frontend/src/components/Post/telescope-post-content.css
> [warn] Code style issues found in the above file(s). Forgot to run Prettier?
> ```

Prettier hated my guts and didn't like the URL then braces on the same line so I had to change the line... PR should still work as intended though

**Review by humphd — APPROVED** [review]

**Review by agarcia-caicedo — APPROVED** [review]
It'll be great to no longer have huge empty spaces at the end of posts

**Pull request merged**